.env.example

```diff
@@ -1,6 +1,8 @@
 # This is an example configuration for the stack.
 # Copy this to `.env` and adapt to your needs.
 
+BASE_URL=localhost
+ENABLE_SSL=False
```

Comment:
SSL must be enabled by default

Reply:
Fixed by commit: https://github.com/nicolas-heigvd/geocity/commit/e52b53831ac54f20aef2094ea4e804b461d2a238